sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3469,6 +3491,40 @@ public partial class EncryptionSettingsGroup
         public static bool operator !=(Azure.ResourceManager.Compute.Models.ExecutionState left, Azure.ResourceManager.Compute.Models.ExecutionState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExpandTypeForListVm : System.IEquatable<Azure.ResourceManager.Compute.Models.ExpandTypeForListVm>
```

Comment:
Could we rename this to `GetVirtualMachinesExpandType`?

Reply:
done